app/controllers/whats_app/webhook_controller.rb

```diff
@@ -73,7 +55,7 @@ def status
 
     def message_params
       params.permit(:AccountSid, :ApiVersion, :Body, :ButtonText, :ButtonPayload, :From, :Latitude, :Longitude,
-                    :MediaContentType0, :MediaUrl0, :MessageSid, :NumMedia, :NumSegments,
+                    :MediaContentType0, :MediaUrl0, :MessageSid, :MessageType, :NumMedia, :NumSegments,
```

Comment:
Its not really related to the PR, just that I like to have a full list of all the params we receive in the payload of the webhooks from third-parties so that if we want to do something new, like find the organization based on some param we receive, we know whether that is possible, or not, without needing to receive the payload again and check its params.